libleptonica/Makefile.tarball

```diff
@@ -0,0 +1,21 @@
+all: installed
+
+TARBALL_VERSION = 1.68
+TARBALL         = build/leptonica-$(TARBALL_VERSION).tar.gz
+#TARBALL_URL     = http://www.leptonica.org/source/leptonica-$(TARBALL_VERSION).tar.gz
+TARBALL_URL     = https://github.com/ipa320/thirdparty/raw/master/leptonica-$(TARBALL_VERSION).tar.gz
+MD5SUM_FILE     = leptonica-$(TARBALL_VERSION).tar.gz.md5sum
+UNPACK_CMD      = tar xzf
+SOURCE_DIR      = build/leptonica-$(TARBALL_VERSION)
+#TARBALL_PATCH   = 
+
+include $(shell rospack find mk)/download_unpack_build.mk
+
+installed: $(SOURCE_DIR)/unpacked
+				cd $(SOURCE_DIR) && ./configure --prefix=`rospack find libleptonica`
```

Comment:
travis seems to be happy, but on my machine ```rospack find libleptonica``` fails in a "virgin" workspace...any ideas?